components/husky_a300/husky_a300_amp_outdoornav_checks.mdx

```diff
@@ -0,0 +1,15 @@
+### Confirming That OutdoorNav Software Is Functioning
+
+Husky AMP includes OutdoorNav Software, which allows the robot to be operated autonomously.
+Follow the steps below to confirm that OutdoorNav Software is functioning.
+
+1. [Connect to Web UI](/docs_outdoornav_user_manual/quick_start#connecting_to_web_ui)
+1. [Check camera views](/docs_outdoornav_user_manual/quick_start#checking-camera-feeds)
+1. [Check system status](/docs_outdoornav_user_manual/quick_start#checking-system-status)
+1. [Check GPS RTK fix](/docs_outdoornav_user_manual/quick_start#checking-gpt-rtk-fix)
+1. Complete system configuration:
+   1. [Load map tiles](/docs_outdoornav_user_manual/quick_start#loading_map_tiles)
+   1. [Set dock location](/docs_outdoornav_user_manual/quick_start#set-dock-location), if a dock is included with your system
+1. [Create your first map](/docs_outdoornav_user_manual/quick_start#create-your-first-map)
+1. [Create your first mission](/docs_outdoornav_user_manual/quick_start#create-your-first-mission)
+1. [Execute your first mission](/docs_outdoornav_user_manual/quick_start#execute-your-first-mission)

```

Comment:
Add a blank line at the end of all MDX files to prevent build / render issues.

Reply:
Added